build/js/live-editor.output_debugger_deps.js

```diff
@@ -1,29 +1,44 @@
-!function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{var f;"undefined"!=typeof window?f=window:"undefined"!=typeof global?f=global:"undefined"!=typeof self&&(f=self),f.iframeOverlay=e()}}(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"./lib/iframe-overlay.js":[function(require,module,exports){
-var Poster = require("../node_modules/poster/lib/poster");
-var EventSim = require("../node_modules/eventsim/lib/eventsim");
-var basic = require("../node_modules/basic-ds/lib/basic");
```

Comment:
most of the changes output_debugger_deps are changes to formatting or new comments.